includes/wikia/controllers/HeliosHelperController.class.php

```diff
@@ -79,6 +88,7 @@ public function sendConfirmationEmail() {
 
 		if ( !$this->wg->EmailAuthentication ) {
 			$this->response->setVal( 'message', 'email authentication is not required' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "message" 16 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)